dashboard/src/components/docs/Moderation.vue

```diff
@@ -0,0 +1,153 @@
+<template>
+  <div>
+    <header>
+      <h1>Moderation</h1>
+      <p>
+        Moderation in Zeppelin is multi-layered. On top of typical actions such
+        as warning, muting, kicking, and banning, Zeppelin allows moderators to
+        utilise flags; create alerts; set thresholds; and act as others.
+      </p>
+      <p>
+        This guide explains the options available in the
+        <router-link to="/docs/plugins/modactions"> Mod actions</router-link>
```

Comment:
Let's move the extra space in the beginning outside of the `router-link` component